ReactAndroid/src/main/java/com/facebook/react/modules/websocket/WebSocketModule.java

```diff
@@ -182,6 +183,22 @@ public void send(String message, int id) {
     }
   }
 
+  @ReactMethod
+  public void sendBinary(String message, int id) {
+    WebSocket client = mWebSocketConnections.get(id);
+    if (client == null) {
+      // This is a programmer error
+      throw new RuntimeException("Cannot send a binary message. Unknown WebSocket id" + id);
+    }
+    try {
+      client.sendMessage(
+        WebSocket.PayloadType.BINARY,
+        new Buffer().write(ByteString.decodeBase64(message)));
+    } catch (IOException | IllegalStateException e) {
```

Comment:
Actually, there's the exact same catch statement in the current code base, see line 182 above. :smile: